README.md

```diff
@@ -11,7 +11,7 @@ In config.js:
 ``` javascript
 System.config({
   map: {
-    "googlmaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!HeinrichFilter/systemjs-plugin-googlemaps/googlemaps"
+    "googlemaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!systemjs-googlemaps"
```

Comment:
Actually I haven't tried it :tongue:

I think we should fix the loading of main anyway... Will look into it later.